Feat: Cloud Gateways network and peering information (#7092)
Kong/docs.konghq.com: app/konnect/network-resiliency.md

keep the connection alive. If it receives no answer, it tries to reconnect to the
control plane after a 5-10 second delay.

> ### How does network peering work with Cloud Gateway nodes?

Reviewer: ```suggestion
### How does network peering work with cloud gateway nodes?
```
Just a note, but I did "dedicated cloud gateway" in lowercase, not sure if that's the way to go, but let me know what you think so we can be consistent between our docs.

Author: The official blog used Dedicated Cloud Gateway, it's not a good data point but I wonder if "Dedicated Cloud Gateway" should be caps, and "cloud gateway" can be lowercase. https://konghq.com/blog/product-releases/dedicated-cloud-gateways

> Each Cloud Gateway node is part of a dedicated Cloud Gateway network that corresponds to a specific cloud region, such as `us-east-1` or `us-west-2`. For enhanced security and seamless connectivity between your cloud network and the {{site.konnect_short_name}} environment, users have the option to peer their Cloud Gateway network with their own network in AWS. This integration is facilitated through the use of AWS's [TransitGateway](https://aws.amazon.com/transit-gateway/) feature, enabling secure network connections across both platforms.

Reviewer: ```suggestion
Each Cloud Gateway node is part of a dedicated Cloud Gateway network that corresponds to a specific cloud region, such as `us-east-1` or `us-west-2`. For enhanced security and seamless connectivity between your cloud network and the {{site.konnect_short_name}} environment, you can peer your Cloud Gateway network with your own network in AWS. This integration is facilitated by using AWS's [TransitGateway](https://aws.amazon.com/transit-gateway/) feature, enabling secure network connections across both platforms.
```
